source/2020-05-15-the-ember-times-issue-148.md

```diff
@@ -25,14 +25,33 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [EmberMap: The `{{on}}` Modifier 🔦](https://embermap.com/topics/what-s-new-in-ember/the-on-modifier-3-11)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Check out the [new EmberMap video](https://twitter.com/ember_map/status/1255510563163197442) on the `{{on}}` modifier, a new way to attach events in Ember. It can be attached in either a plain HTML element or an Ember component.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+```hbs
+<Toggle
+  @checked={{this.isAdmin}}
+  @onChange={{this.toggleIsAdmin}}
+  {{on 'mouseenter' this.showTip}}
+  {{on 'mouseleave' this.hideTip}}
+/>
+```
+
+The `{{on}}` modifier accepts two arguments: the event name and the function handler. The **event name** is actually just the same argument that the vanilla JS API `element.addEventListener(eventName)` accepts. And the **function handler** is any action, meaning any properly bound function. It can either live on the component instance or be passed in as an argument.
```

Comment:
The Alex lint suggested not using "just." I think, in your sentence, we can omit the word.

```suggestion
The `{{on}}` modifier accepts two arguments: the event name and the function handler. The **event name** is actually the same argument that the vanilla JS API `element.addEventListener(eventName)` accepts. And the **function handler** is any action, meaning any properly bound function. It can either live on the component instance or be passed in as an argument.
```

Reply:
Thank you!